### PR TITLE
Ember CLI 2016-04-21

### PR DESCRIPTION
**Pull request by nathanhammond**
Respond with reactions as to accuracy. Will be merged without objection once I remember to do so.

/cc @stefanpenner @hjdivad @trabus @kellyselden @kategengler @Turbo87

### CODE REVIEW

File: ember-cli/2016-04/april-21.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Discussion Notes
+
+### ember module migrator
+
+- [https://github.com/rwjblue/ember-module-migrator](https://github.com/rwjblue/ember-module-migrator)
+- Single purpose, migrates from existing structures to new modules structure
+- Currently only does file-rename for classic structure.
+- Working on pod structure, and mixed structure.
+
+### Updates on <base> (Tobias)
```
Comment (Turbo87) [inline ember-cli/2016-04/april-21.md]: `<base>` isn't rendering... 😉

File: ember-cli/2016-04/april-21.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+- Mostly ready to be merged.
+- Seems like it works.
+- Push into Canary (2.7) and see what happens.
+
+### Private pre-concat hook(s) (Nathan)
+
+- Open RFC
+- Multiple times vs. one time, which trees, and more.
+- Possibly slow to expose.
```
Comment (stefanpenner) [inline ember-cli/2016-04/april-21.md]: @chadhietala would love you to keep an eye on this one.

File: ember-cli/2016-04/april-21.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+- ember-cli betas, do they include stable ember by default, or also ember betas? 
+  - general seems to believe we should stick to stable releases of ember.
+  - release final ember-cli beta with new stable releases of ember and ember-data?
+
+### YUIDoc output
+
+- Publish the YUIDocs to ember-cli.com/api? **Yes.**
+- Versioned docs will come later–one step at  a time.
+  - Can we rope in the learning team?
```
Comment (locks) [inline ember-cli/2016-04/april-21.md]: yes please :P

File: ember-cli/2016-04/april-21.md (hunk 2 above)
Comment (chadhietala) [inline ember-cli/2016-04/april-21.md]: 😬

**Pull request merged**